sql/mob_spawn_points.sql

```diff
@@ -3180,9 +3180,9 @@ INSERT INTO `mob_spawn_points` VALUES (16842757,'Wanderer','Wanderer',1,-32.046,
 INSERT INTO `mob_spawn_points` VALUES (16842758,'Weeper','Weeper',2,-48.354,-0.500,114.108,96);
 INSERT INTO `mob_spawn_points` VALUES (16842759,'Wanderer','Wanderer',1,-80.057,-0.500,122.893,219);
 INSERT INTO `mob_spawn_points` VALUES (16842760,'Wanderer','Wanderer',1,-82.961,-0.500,119.686,139);
-INSERT INTO `mob_spawn_points` VALUES (16842761,'Wanderer','Wanderer',1,-69.819,-0.500,123.495,248);
-INSERT INTO `mob_spawn_points` VALUES (16842762,'Weeper','Weeper',2,-80.403,-0.500,123.475,6);
-INSERT INTO `mob_spawn_points` VALUES (16842763,'Weeper','Weeper',2,-69.238,-0.500,124.049,73);
+INSERT INTO `mob_spawn_points` VALUES (16842761,'Wanderer','Wanderer',1,-6.822,-0.500,155.388,248);
```

Comment:
Do the position edits in this file come from captures?

Reply:
Reverted pos changes as they were not from captures.